code/script.js

```diff
@@ -0,0 +1,88 @@
+const owner = 'joannaringqvist';
+const API_URL_REPOS = `https://api.github.com/users/${owner}/repos`;
+const projects = document.getElementById('projects');
+const username = document.getElementById('username');
+const picture = document.getElementById('picture');
+const numberOfProjects = document.getElementById('numberOfProjects');
+const sort = document.getElementById('sort');
+let ownerLogin = '';
+
+//Function for getting all my repos from Github
+const getRepos = (sort) => {
+    fetch(API_URL_REPOS)
+        .then((res) => res.json())
+        .then((data) => {
+
+            //Get the name and picture from the user and pass it to the function for showing them
+            ownerLogin = data[0].owner.login;
+            userPic = data[0].owner.avatar_url;
+            showUsernameAndPic(ownerLogin, userPic);
+            
+            //Array for storing the repos and counting them
+            let arrayWithRepos = [];
+
+            data.forEach((repo) => {
+                //I only want to continue working with the repos from Technigo, so they should be forked and start with project
+                if (repo.fork === true && repo.name.startsWith('project')) {
+                    arrayWithRepos.push(repo.name);
+                    //Write the table rows and cells
+                    projects.innerHTML += `
+                    <tr>
+                        <td>${repo.name}</td>
+                        <td>${new Date(repo.updated_at).toLocaleDateString('sv-SE')}</td>
+                        <td>${repo.default_branch}</td>
+                        <td id="commits-${repo.name}"></td>
+                        <td><a class="repo-url" href="${repo.html_url}">${repo.html_url}</a></td>
```

Comment:
A really small comment, but here is the only time double quotation marks are used. To make the code consistent only single quotation marks (or double) should be used.